ci/prow/config.yaml

```diff
@@ -214,6 +214,39 @@ presubmits:
           requests:
             memory: "1Gi"
 
+  - name: pull-knative-serving-upgrade-tests
```

Comment:
Also, do we want to run these tests in the CI flow?

Reply:
I think so -- I'm not sure if we want them to be fatal yet, @mattmoor thoughts?

Reply:
Matt says yes to CI flow and yes to failures being fatal (i.e. must pass).

Reply:
Ok. https://github.com/knative/serving/pull/2502 will make the upgrade test run on CI and presubmits. Then we'll just have to tweak the Prow jobs and we're done.

Reply:
On line 209, please replace `- "--integration-tests"` with:

```
        - "--run-test"
        - "./test/e2e-tests.sh"
```